go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -304,7 +303,7 @@ func postErsCompletion(topologyRecovery *TopologyRecovery, analysisEntry *inst.R
 		_ = AuditTopologyRecovery(topologyRecovery, message)
 		_ = inst.AuditOperation(recoveryName, analysisEntry.AnalyzedInstanceAlias, message)
 		_ = AuditTopologyRecovery(topologyRecovery, fmt.Sprintf("%v: successfully promoted %+v", recoveryName, promotedReplica.InstanceAlias))
-		vtopsExec.RaiseProblem(analysisEntry.AnalyzedInstanceHostname, "orc-dead-tablet")
+		vtopsExec.RaiseProblem(analysisEntry, topologyRecovery.SuccessorAlias, "orc-dead-tablet")
```

Comment:
Isn't this raising the problem for the wrong host? The successor is the newly promoted one, but we want the failed one to get the problem.

Reply:
@edwardIshaq no, the `SuccessorAlias` is used to know the primary, the host we reparented is in `analysisEntry`

Reply:
ahh, okay, so the problem is still raised against the old/dead primary.. and we changed the api with specifying the new/current primary..